go.mod

```diff
@@ -219,22 +220,23 @@ replace (
 	github.com/client9/misspell => github.com/client9/misspell v0.3.4
 	github.com/cloudflare/cfssl => github.com/cloudflare/cfssl v0.0.0-20180726162950-56268a613adf
 	github.com/clusterhq/flocker-go => github.com/clusterhq/flocker-go v0.0.0-20160920122132-2b8b7259d313
+	github.com/cockroachdb/datadriven => github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa
 	github.com/codegangsta/negroni => github.com/codegangsta/negroni v1.0.0
 	github.com/container-storage-interface/spec => github.com/container-storage-interface/spec v1.1.0
 	github.com/containerd/console => github.com/containerd/console v0.0.0-20170925154832-84eeaae905fa
 	github.com/containerd/containerd => github.com/containerd/containerd v1.0.2
 	github.com/containerd/typeurl => github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20
 	github.com/containernetworking/cni => github.com/containernetworking/cni v0.7.1
 	github.com/coredns/corefile-migration => github.com/coredns/corefile-migration v1.0.2
-	github.com/coreos/bbolt => github.com/coreos/bbolt v1.3.1-coreos.6
-	github.com/coreos/etcd => github.com/coreos/etcd v3.3.15+incompatible
+	github.com/coreos/etcd => github.com/coreos/etcd v3.3.10+incompatible
```

Comment:
if we're still going to list this, let's keep it at the same level we were at previously (v3.3.15)

Reply:
@liggitt, I tried to keep github.com/coreos/etcd at v3.3.15, but the verify-vendor keeps wanting me to pin it back to 3.3.10. 

```
github.com/coreos/etcd
    pinned:    v3.3.15+incompatible
    preferred: v3.3.10+incompatible
    hack/pin-dependency.sh github.com/coreos/etcd v3.3.10+incompatible
```

I wonder if this is related to what you mentioned in here: https://github.com/kubernetes/kubernetes/pull/81434#issuecomment-522673849

```
github.com/spf13/viper@v1.3.2 github.com/coreos/go-etcd@v2.0.0+incompatible
github.com/spf13/viper@v1.3.2 github.com/coreos/etcd@v3.3.10+incompatible
```
How or where could I see this and how to resolve this "transitive dependencies on any semantically-tagged etcd versions."?